lib/paper_trail/has_paper_trail.rb

```diff
@@ -192,7 +192,7 @@ def paper_trail_originator
       end
 
       def originator
-        warn "DEPRECATED: use `paper_trail_originator` instead of `originator`. Support for `originator` will be removed in PaperTrail 4.0"
+        ActiveSupport::Deprecation.warn "Use paper_trail_originator instead of originator."
```

Comment:
Let's fully qualify these constants by prepending `::` just in case we later add an `ActiveSupport` module within the gem.  (`::ActiveSupport::Deprecation`).